### PR TITLE
[BPK-2472] Run tests on testlab

### PR DESCRIPTION
**Pull request by tiagoengel**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:4aa4f11a26e26eedf2ca451a411451833fd1f336`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:d620cbf3630c389d34b2ff4953d88c1db06721cd`

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: android
-sudo: false
+sudo: required
```
Comment (tiagoengel) [inline .travis.yml]: Not actually required but gives us more memory!

File: build.gradle [hunk 2]
```diff
@@ -5,13 +5,14 @@ buildscript {
...
     jcenter()
+    maven { url 'https://jitpack.io' }
   }
   dependencies {
     classpath 'com.android.tools.build:gradle:3.2.1'
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
     classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
-    classpath 'com.facebook.testing.screenshot:plugin:0.8.0'
+    classpath 'com.github.tiagoengel:screenshot-tests-for-android:fix-reference-dir-SNAPSHOT'
```
Comment (tiagoengel) [inline build.gradle]: Branch with the fix to just compare screenshots and not record them (we do that on test lab now). I'll try to get this merged back into the main branch in due time.

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:d4e804af94c595845a4915ef25739c07a04d7690`

**Pull request merged**